src/qml/editorwidgets/ValueMap.qml

```diff
@@ -198,7 +321,7 @@ EditorWidgetBase {
                 searchFeaturePopup.close()
               }
             }
-          }
+                          }
```

Comment:
@mohsenD98 , sorry, one last thing, I think there is an indentation issue here and on another line below?

Reply:
Ah, I apologize for the mistake. I've corrected the formatting now (problems with QML/Js auto formatting).